storage/google/cloud/storage/blob.py

```diff
@@ -348,28 +370,23 @@ def download_to_file(self, file_obj, client=None):
         :raises: :class:`google.cloud.exceptions.NotFound`
         """
         client = self._require_client(client)
-        if self.media_link is None:  # not yet loaded
-            self.reload()
-
-        download_url = self.media_link
-
-        # Use apitools 'Download' facility.
-        download = Download.from_stream(file_obj)
-
-        if self.chunk_size is not None:
-            download.chunksize = self.chunk_size
-
+        # Get the download URL.
+        download_url = self._get_download_url()
```

Comment:
That's correct, the docstring of `_get_download_url()` explains why not:

>  If the ``media_link`` has been loaded, it will be used, otherwise the URL will be constructed from the current blob's path (and possibly generation) to avoid a round trip.